pkg/addon/parameters_test.go

```diff
@@ -61,21 +85,39 @@ func TestGetParameter(t *testing.T) {
 					},
 				},
 			}
+			namespace := &corev1.Namespace{
+				ObjectMeta: v1.ObjectMeta{
+					Name: testOperatorNamespace,
+				},
+			}
+			subscription := &v1alpha1.Subscription{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      addonName,
+					Namespace: testOperatorNamespace,
+				},
+			}
```

Comment:
We could setup client and init objects before this moment, but It would make it awkward to modify those init objects depending on the scenario we want to test :thinking: 
As for the `rhoami` - the only difference would be an operator namespace here. As far as I know, we have Internal RHOAM only installed as an add-on. In this case, it will find the secret using the same logic as if it would be RHOAM, so there is no need for an additional scenario. WDYT?

Reply:
If it makes it awkward to to modify these objects when moved I think it's ok to leave them here.

As for a scenario that is testing rhoami. Yes, we should have one. This has caused issues before. It is more about proving works and that it keeps working.

Reply:
now we will be testing for `redhat-rhoam-operator` namespace and I've added one extra case for `rhoami` namespace that would ask to extract parameters from the secret